File: about.go
package main

import tb "gopkg.in/tucnak/telebot.v2"

func (gl *Global) handleAbout(m *tb.Message) {
	output := "*ManDown*:\n\n" +
		"Open Source on [GitHub](https://github.com/Donnie/ManDown)\n" +
		"Hosted on Vultr.com in New Jersey, USA\n" +
		"No personally identifiable information is stored or used by this bot."

	gl.Bot.Send(m.Sender, output)
}

File: src/about.rs
use teloxide::{prelude::*, types::ParseMode};

pub async fn handle_about(bot: Bot, msg: Message) -> ResponseResult<()> {
  let output = "<b>ManDown</b>:
Open Source on <a href='https://github.com/Donnie/ManDown'>GitHub</a>
Hosted on Vultr.com in New Jersey, USA
No personally identifiable information is stored or used by this bot.";

  bot.send_message(msg.chat.id, output).parse_mode(ParseMode::Html).await?;

  Ok(())
}
